### PR TITLE
[Git] New issue templates

### PR DESCRIPTION
**Pull request by ronijaakkola**
## Description

Added new issue templates and updated old ones to match HDS needs. New issue templates automatically apply corresponding labels to issues.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,38 +1,43 @@
...
- - Version [e.g. 22]
+## Device information
+**Desktop (please complete the following information)**:
+>  - OS: [e.g. iOS]
+> - Browser [e.g. chrome, safari]
+>  - Version [e.g. 22]
 
-**Additional context**
-Add any other context about the problem here.
+**martphone (please complete the following information)**:
```
Comment (niglu1) [inline .github/ISSUE_TEMPLATE/bug_report.md:36]: typo > Smartphone

File: .github/ISSUE_TEMPLATE/enhancement.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Enhancement
+about: Request a change to existing component, pattern, token or documentation.
```
Comment (niglu1) [inline .github/ISSUE_TEMPLATE/enhancement.md:3]: should probably be "**an** existing"

**Comment by niglu1** [comment]
Sorry, I completely missed this!

**Comment by ronijaakkola** [comment]
@niglu1 Pushed fixes to those typos.

**Pull request merged**